vulns.go

```diff
@@ -125,6 +172,22 @@ Create a list of issue details without tickets.
 ***/
 func getSnykOpenSourceIssueWithoutTickets(flags flags, projectID string, maturityFilter []string, tickets map[string]string, customDebug debug, responseAggregatedData []byte) (map[string]interface{}, string, error) {
 
+    issueType := flags.optionalFlags.issueType
```

Comment:
This issue is now fixed. There is no need for those changes. Please revert.

Reply:
@mathild3r You have fixed license issue parsing, but still it is going to return both `vuln` and `license` type issue irrespective of `--type` flag. While I have added a check to return issues only of type user has requested using `--type` flag, if no `--type` flag then return all type of issues, can you please review again?
I'll resolve conflicts now.

Reply:
I do some manual testing next week to check that, we filter by issue type when querying the endpoint to get the vulnerabilities (see line 101) so we shouldn't receive anything that is not of the expected type.